rust/libnewsboat/tests/resolve_relative.rs

```diff
@@ -0,0 +1,14 @@
+extern crate libnewsboat;
+
+use libnewsboat::utils;
+use std::env;
+use std::path::Path;
+
+#[test]
+fn t_resolve_relative() {
```

Comment:
I think this can be moved into a unit-test in `tests` module in rust/libnewsboat/src/utils.rs because `utils::resolve_relative` doesn't have side effects, doesn't need any state, and doesn't contend on any resources with any other test.